tests/plugin_tests/cve/fixtures.py

```diff
@@ -7,6 +8,18 @@
 from nbdefense.plugins.cve.cve_plugin import CVEPlugin
 
 
+@pytest.fixture(scope="session")
+def create_cve_requirements_file_path() -> Path:
+    current_directory = Path(os.path.dirname(os.path.realpath(__file__)))
```

Comment:
We could use a [tempfile](https://docs.python.org/3/library/tempfile.html) instead of creating a file in the working directory.